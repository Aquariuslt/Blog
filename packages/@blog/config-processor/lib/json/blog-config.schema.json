{
  "$schema": "http://json-schema.org/draft-07/schema",
  "$id": "http://github.com/aquariuslt/blog/config.schema.json",
  "title": "Blog Configuration",
  "description": "blog config file yml specification",
  "type": "object",
  "properties": {
    "version": {
      "type": "string",
      "default": "1.0.0"
    },
    "site": {
      "$ref": "#/definitions/site"
    },
    "build": {
      "$ref": "#/definitions/build"
    },
    "features": {
      "$ref": "#/definitions/features"
    }
  },
  "required": ["version", "site", "build"],
  "definitions": {
    "site": {
      "type": "object",
      "properties": {
        "host": {
          "type": "string",
          "format": "url",
          "description": "Your host, e.g. http://your-site.com or https://my-site.com."
        },
        "title": {
          "type": "string",
          "description": "Your blog title."
        },
        "author": {
          "type": "string",
          "description": "Your name"
        },
        "avatar": {
          "type": "string",
          "description": "Your profile picture"
        },
        "subtitle": {
          "type": "string",
          "description": "Your blog sub title"
        },
        "description": {
          "type": "string",
          "description": "Your blog description, this usually will built in html head meta"
        },
        "language": {
          "type": "string",
          "description": "Your prefer language",
          "enum": ["zh", "en"],
          "default": "zh"
        },
        "copyright": {
          "type": "string",
          "description": "Site copyright, usually appear under your website",
          "default": "copyright 2019 aquariuslt blog"
        }
      },
      "required": ["host", "title", "subtitle", "description"]
    },
    "build": {
      "type": "object",
      "properties": {
        "directory": {
          "type": "object",
          "properties": {
            "posts": {
              "type": "string",
              "description": "markdown posts file base directory, relative to project root path",
              "default": "posts"
            },
            "public": {
              "type": "string",
              "description": "build location path",
              "default": "dist"
            }
          },
          "required": []
        },
        "favicon": {
          "type": "string",
          "description": "Your favicon file location",
          "default": "favicon.ico"
        },
        "theme": {
          "type": "string",
          "description": "Your blog theme package name, not implementation now",
          "default": "vue"
        },
        "colors": {
          "type": "object",
          "description": "Your blog theme color preset from material color customization",
          "default": {}
        }
      },
      "required": []
    },
    "features": {
      "type": "object",
      "properties": {
        "ga": {
          "type": "string",
          "description": "Your Google Analytics tracking id"
        },
        "disqus": {
          "type": "string",
          "description": "Your disqus short id"
        }
      },
      "required": []
    }
  }
}
