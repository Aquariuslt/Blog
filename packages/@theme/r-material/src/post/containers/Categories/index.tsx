import * as React from 'react';

type CategoriesProps = BlogApiModel.CategoriesOverview;

const Categories: React.ComponentType<CategoriesProps> = (props: CategoriesProps) => {
  return <div>Categories</div>;
};

export default Categories;
