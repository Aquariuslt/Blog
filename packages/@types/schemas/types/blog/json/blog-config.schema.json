{
  "$schema": "http://json-schema.org/draft-07/schema",
  "$id": "http://github.com/aquariuslt/blog/config.schema.json",
  "description": "blog config file yml specification",
  "type": "object",
  "properties": {
    "version": {
      "type": "string",
      "default": "1.0.0"
    },
    "site": {
      "$ref": "#/definitions/site"
    },
    "build": {
      "$ref": "#/definitions/build"
    }
  },
  "required": [
    "version",
    "site",
    "build"
  ],
  "definitions": {
    "site": {
      "type": "object",
      "properties": {
        "host": {
          "type": "string",
          "format": "url"
        },
        "title": {
          "type": "string"
        },
        "subtitle": {
          "type": "string"
        },
        "description": {
          "type": "string"
        },
        "language": {
          "type": "string",
          "enum": [
            "zh",
            "en"
          ],
          "default": "zh"
        },
        "copyright": {
          "type": "string",
          "default": "copyright 2019 aquariuslt blog"
        }
      },
      "required": [
        "host",
        "title",
        "subtitle",
        "description"
      ]
    },
    "build": {
      "type": "object",
      "properties": {
        "directory": {
          "type": "object",
          "properties": {
            "posts": {
              "type": "string",
              "description": "markdown posts file base directory, relative to project root path",
              "default": "posts"
            },
            "public": {
              "type": "string",
              "description": "build location path",
              "default": "dist"
            }
          },
          "required": []
        },
        "favicon": {
          "type": "string",
          "default": "favicon.ico"
        },
        "theme": {
          "type": "string",
          "default": "vue"
        }
      },
      "required": []
    }
  }
}
