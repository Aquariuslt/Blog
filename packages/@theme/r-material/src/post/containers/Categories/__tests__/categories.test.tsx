import * as React from 'react';
import * as Enzyme from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import { BrowserRouter as Router } from 'react-router-dom';

import Categories from '../';

import categoryStore from '../../../stores/category.store';

import * as mockCategoriesOverview from './__mocks__/sample-categories.json';

Enzyme.configure({ adapter: new Adapter() });

describe('@theme/r-material: post/container: Categories', () => {
  it('# should call load categories overview at mount', () => {
    const mockMatch = {
      params: {},
      isExact: true,
      path: '/categories',
      url: '/categories'
    };

    categoryStore.categories = mockCategoriesOverview;

    const wrapper = Enzyme.mount(
      <Router>
        <Categories match={mockMatch} categoryStore={categoryStore} />
      </Router>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
