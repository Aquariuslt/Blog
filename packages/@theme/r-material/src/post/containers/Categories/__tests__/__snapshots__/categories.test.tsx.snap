// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@theme/r-material: post/container: Categories # should call load categories overview at mount 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <inject-with-categoryStore(Object)
      categoryStore={
        CategoryStore {
          "categories": Array [
            Object {
              "link": "/categories/Blog",
              "name": "Blog",
              "total": 13,
            },
            Object {
              "link": "/categories/Others",
              "name": "Others",
              "total": 3,
            },
            Object {
              "link": "/categories/Note",
              "name": "Note",
              "total": 2,
            },
            Object {
              "link": "/categories/Debug",
              "name": "Debug",
              "total": 3,
            },
          ],
        }
      }
      match={
        Object {
          "isExact": true,
          "params": Object {},
          "path": "/categories",
          "url": "/categories",
        }
      }
    >
      <Memo(wrappedComponent)
        categoryStore={
          CategoryStore {
            "categories": Array [
              Object {
                "link": "/categories/Blog",
                "name": "Blog",
                "total": 13,
              },
              Object {
                "link": "/categories/Others",
                "name": "Others",
                "total": 3,
              },
              Object {
                "link": "/categories/Note",
                "name": "Note",
                "total": 2,
              },
              Object {
                "link": "/categories/Debug",
                "name": "Debug",
                "total": 3,
              },
            ],
          }
        }
        match={
          Object {
            "isExact": true,
            "params": Object {},
            "path": "/categories",
            "url": "/categories",
          }
        }
      >
        <WithStyles(ForwardRef(Paper))
          className="makeStyles-root-1"
        >
          <ForwardRef(Paper)
            className="makeStyles-root-1"
            classes={
              Object {
                "elevation0": "MuiPaper-elevation0",
                "elevation1": "MuiPaper-elevation1",
                "elevation10": "MuiPaper-elevation10",
                "elevation11": "MuiPaper-elevation11",
                "elevation12": "MuiPaper-elevation12",
                "elevation13": "MuiPaper-elevation13",
                "elevation14": "MuiPaper-elevation14",
                "elevation15": "MuiPaper-elevation15",
                "elevation16": "MuiPaper-elevation16",
                "elevation17": "MuiPaper-elevation17",
                "elevation18": "MuiPaper-elevation18",
                "elevation19": "MuiPaper-elevation19",
                "elevation2": "MuiPaper-elevation2",
                "elevation20": "MuiPaper-elevation20",
                "elevation21": "MuiPaper-elevation21",
                "elevation22": "MuiPaper-elevation22",
                "elevation23": "MuiPaper-elevation23",
                "elevation24": "MuiPaper-elevation24",
                "elevation3": "MuiPaper-elevation3",
                "elevation4": "MuiPaper-elevation4",
                "elevation5": "MuiPaper-elevation5",
                "elevation6": "MuiPaper-elevation6",
                "elevation7": "MuiPaper-elevation7",
                "elevation8": "MuiPaper-elevation8",
                "elevation9": "MuiPaper-elevation9",
                "root": "MuiPaper-root",
                "rounded": "MuiPaper-rounded",
              }
            }
          >
            <div
              className="MuiPaper-root MuiPaper-elevation1 makeStyles-root-1 MuiPaper-rounded"
            >
              <WithStyles(ForwardRef(Table))
                className="makeStyles-table-2"
              >
                <ForwardRef(Table)
                  className="makeStyles-table-2"
                  classes={
                    Object {
                      "root": "MuiTable-root",
                    }
                  }
                >
                  <table
                    className="MuiTable-root makeStyles-table-2"
                  >
                    <WithStyles(ForwardRef(TableHead))>
                      <ForwardRef(TableHead)
                        classes={
                          Object {
                            "root": "MuiTableHead-root",
                          }
                        }
                      >
                        <thead
                          className="MuiTableHead-root"
                        >
                          <WithStyles(ForwardRef(TableRow))>
                            <ForwardRef(TableRow)
                              classes={
                                Object {
                                  "footer": "MuiTableRow-footer",
                                  "head": "MuiTableRow-head",
                                  "hover": "MuiTableRow-hover",
                                  "root": "MuiTableRow-root",
                                  "selected": "Mui-selected",
                                }
                              }
                            >
                              <tr
                                className="MuiTableRow-root MuiTableRow-head"
                              >
                                <WithStyles(ForwardRef(TableCell))
                                  align="left"
                                >
                                  <ForwardRef(TableCell)
                                    align="left"
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                  >
                                    <th
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft"
                                      scope="col"
                                    >
                                      post.navigation.category.label
                                    </th>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                                <WithStyles(ForwardRef(TableCell))
                                  align="right"
                                >
                                  <ForwardRef(TableCell)
                                    align="right"
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                  >
                                    <th
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
                                      scope="col"
                                    >
                                      post.navigation.total.label
                                    </th>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                              </tr>
                            </ForwardRef(TableRow)>
                          </WithStyles(ForwardRef(TableRow))>
                        </thead>
                      </ForwardRef(TableHead)>
                    </WithStyles(ForwardRef(TableHead))>
                    <WithStyles(ForwardRef(TableBody))>
                      <ForwardRef(TableBody)
                        classes={
                          Object {
                            "root": "MuiTableBody-root",
                          }
                        }
                      >
                        <tbody
                          className="MuiTableBody-root"
                        >
                          <WithStyles(ForwardRef(TableRow))
                            className="makeStyles-row-3"
                            hover={true}
                            key="Blog"
                          >
                            <ForwardRef(TableRow)
                              className="makeStyles-row-3"
                              classes={
                                Object {
                                  "footer": "MuiTableRow-footer",
                                  "head": "MuiTableRow-head",
                                  "hover": "MuiTableRow-hover",
                                  "root": "MuiTableRow-root",
                                  "selected": "Mui-selected",
                                }
                              }
                              hover={true}
                            >
                              <tr
                                className="MuiTableRow-root makeStyles-row-3 MuiTableRow-hover"
                              >
                                <WithStyles(ForwardRef(TableCell))
                                  component="th"
                                  scope="row"
                                >
                                  <ForwardRef(TableCell)
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                    component="th"
                                    scope="row"
                                  >
                                    <th
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-body"
                                      scope="row"
                                    >
                                      <Link
                                        className="makeStyles-link-4"
                                        to="/categories/Blog"
                                      >
                                        <a
                                          className="makeStyles-link-4"
                                          href="/categories/Blog"
                                          onClick={[Function]}
                                        >
                                          Blog
                                        </a>
                                      </Link>
                                    </th>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                                <WithStyles(ForwardRef(TableCell))
                                  align="right"
                                >
                                  <ForwardRef(TableCell)
                                    align="right"
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                  >
                                    <td
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                                    >
                                      13
                                    </td>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                              </tr>
                            </ForwardRef(TableRow)>
                          </WithStyles(ForwardRef(TableRow))>
                          <WithStyles(ForwardRef(TableRow))
                            className="makeStyles-row-3"
                            hover={true}
                            key="Others"
                          >
                            <ForwardRef(TableRow)
                              className="makeStyles-row-3"
                              classes={
                                Object {
                                  "footer": "MuiTableRow-footer",
                                  "head": "MuiTableRow-head",
                                  "hover": "MuiTableRow-hover",
                                  "root": "MuiTableRow-root",
                                  "selected": "Mui-selected",
                                }
                              }
                              hover={true}
                            >
                              <tr
                                className="MuiTableRow-root makeStyles-row-3 MuiTableRow-hover"
                              >
                                <WithStyles(ForwardRef(TableCell))
                                  component="th"
                                  scope="row"
                                >
                                  <ForwardRef(TableCell)
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                    component="th"
                                    scope="row"
                                  >
                                    <th
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-body"
                                      scope="row"
                                    >
                                      <Link
                                        className="makeStyles-link-4"
                                        to="/categories/Others"
                                      >
                                        <a
                                          className="makeStyles-link-4"
                                          href="/categories/Others"
                                          onClick={[Function]}
                                        >
                                          Others
                                        </a>
                                      </Link>
                                    </th>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                                <WithStyles(ForwardRef(TableCell))
                                  align="right"
                                >
                                  <ForwardRef(TableCell)
                                    align="right"
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                  >
                                    <td
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                                    >
                                      3
                                    </td>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                              </tr>
                            </ForwardRef(TableRow)>
                          </WithStyles(ForwardRef(TableRow))>
                          <WithStyles(ForwardRef(TableRow))
                            className="makeStyles-row-3"
                            hover={true}
                            key="Note"
                          >
                            <ForwardRef(TableRow)
                              className="makeStyles-row-3"
                              classes={
                                Object {
                                  "footer": "MuiTableRow-footer",
                                  "head": "MuiTableRow-head",
                                  "hover": "MuiTableRow-hover",
                                  "root": "MuiTableRow-root",
                                  "selected": "Mui-selected",
                                }
                              }
                              hover={true}
                            >
                              <tr
                                className="MuiTableRow-root makeStyles-row-3 MuiTableRow-hover"
                              >
                                <WithStyles(ForwardRef(TableCell))
                                  component="th"
                                  scope="row"
                                >
                                  <ForwardRef(TableCell)
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                    component="th"
                                    scope="row"
                                  >
                                    <th
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-body"
                                      scope="row"
                                    >
                                      <Link
                                        className="makeStyles-link-4"
                                        to="/categories/Note"
                                      >
                                        <a
                                          className="makeStyles-link-4"
                                          href="/categories/Note"
                                          onClick={[Function]}
                                        >
                                          Note
                                        </a>
                                      </Link>
                                    </th>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                                <WithStyles(ForwardRef(TableCell))
                                  align="right"
                                >
                                  <ForwardRef(TableCell)
                                    align="right"
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                  >
                                    <td
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                                    >
                                      2
                                    </td>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                              </tr>
                            </ForwardRef(TableRow)>
                          </WithStyles(ForwardRef(TableRow))>
                          <WithStyles(ForwardRef(TableRow))
                            className="makeStyles-row-3"
                            hover={true}
                            key="Debug"
                          >
                            <ForwardRef(TableRow)
                              className="makeStyles-row-3"
                              classes={
                                Object {
                                  "footer": "MuiTableRow-footer",
                                  "head": "MuiTableRow-head",
                                  "hover": "MuiTableRow-hover",
                                  "root": "MuiTableRow-root",
                                  "selected": "Mui-selected",
                                }
                              }
                              hover={true}
                            >
                              <tr
                                className="MuiTableRow-root makeStyles-row-3 MuiTableRow-hover"
                              >
                                <WithStyles(ForwardRef(TableCell))
                                  component="th"
                                  scope="row"
                                >
                                  <ForwardRef(TableCell)
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                    component="th"
                                    scope="row"
                                  >
                                    <th
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-body"
                                      scope="row"
                                    >
                                      <Link
                                        className="makeStyles-link-4"
                                        to="/categories/Debug"
                                      >
                                        <a
                                          className="makeStyles-link-4"
                                          href="/categories/Debug"
                                          onClick={[Function]}
                                        >
                                          Debug
                                        </a>
                                      </Link>
                                    </th>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                                <WithStyles(ForwardRef(TableCell))
                                  align="right"
                                >
                                  <ForwardRef(TableCell)
                                    align="right"
                                    classes={
                                      Object {
                                        "alignCenter": "MuiTableCell-alignCenter",
                                        "alignJustify": "MuiTableCell-alignJustify",
                                        "alignLeft": "MuiTableCell-alignLeft",
                                        "alignRight": "MuiTableCell-alignRight",
                                        "body": "MuiTableCell-body",
                                        "footer": "MuiTableCell-footer",
                                        "head": "MuiTableCell-head",
                                        "paddingCheckbox": "MuiTableCell-paddingCheckbox",
                                        "paddingNone": "MuiTableCell-paddingNone",
                                        "root": "MuiTableCell-root",
                                        "sizeSmall": "MuiTableCell-sizeSmall",
                                      }
                                    }
                                  >
                                    <td
                                      aria-sort={null}
                                      className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
                                    >
                                      3
                                    </td>
                                  </ForwardRef(TableCell)>
                                </WithStyles(ForwardRef(TableCell))>
                              </tr>
                            </ForwardRef(TableRow)>
                          </WithStyles(ForwardRef(TableRow))>
                        </tbody>
                      </ForwardRef(TableBody)>
                    </WithStyles(ForwardRef(TableBody))>
                  </table>
                </ForwardRef(Table)>
              </WithStyles(ForwardRef(Table))>
            </div>
          </ForwardRef(Paper)>
        </WithStyles(ForwardRef(Paper))>
      </Memo(wrappedComponent)>
    </inject-with-categoryStore(Object)>
  </Router>
</BrowserRouter>
`;
